Branch-SDK/Branch-SDK/BNCEncodingUtils.m

```diff
@@ -197,10 +197,10 @@ + (NSString *)iso8601StringFromDate:(NSDate *)date {
 
 + (NSString *)sanitizedStringFromString:(NSString *)dirtyString {
     NSString *cleanString = [[[[dirtyString stringByReplacingOccurrencesOfString:@"\"" withString:@"\\\""]
-                                            stringByReplacingOccurrencesOfString:@"\n" withString:@"\\n"]
-                                            stringByReplacingOccurrencesOfString:@"’" withString:@"'"]
-                                            stringByReplacingOccurrencesOfString:@"\r" withString:@"\\r"];
-
+                               stringByReplacingOccurrencesOfString:@"\n" withString:@"\\n"]
+                              stringByReplacingOccurrencesOfString:@"’" withString:@"'"]
+                             stringByReplacingOccurrencesOfString:@"\r" withString:@"\\r"];
+    
```

Comment:
minor - do you mind either modifying your editor to be consistent with the whitespace here or unchecking this specific file in? looks like nothing changed